docs/rfc/000-Payload-v37.md

```diff
@@ -24,8 +24,11 @@ type PayloadAlpha37 = [
   // we can save more space
   authorNetwork: UTF8String | SocialNetworkEnum,
   authorID: UTF8String,
-  authorPublicKey: ArrayBuffer,
-  // authorPublicKeyCurve: ???,
+  authorPublicKey: ArrayBuffer,                 // format: [PublicKey, SupportedCurvesEnum]
+  // array of base58 strings of newly generated ephemeral public keys
+  // indexed by SupportedCurves
+  authorEphemeralPublicKeyList: string[],       // length = Object.keys(SupportedCurves).length 
```

Comment:
```suggestion
  authorEphemeralPublicKeyList: SupportedCurve[],       // the list is unique?
```

Reply:
This is not how it works. This list consists of a bunch of public keys not enums.